src/Emulator/Peripherals/Peripherals.csproj

```diff
@@ -446,5 +450,6 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="Peripherals\Miscellaneous\SiLabs\" />
+    <Compile Include="Peripherals\Timers\STM32F4_Watchdog.cs" />
```

Comment:
please drop all changes to this file and leave only this line adding the file to the project - but also please move it to the section around lines 50-390, where all the other files are located.